.github/workflows/gsdump-workflow.yml

```
@@ -0,0 +1,97 @@
+name: GSDump Comparer
+on: 
+  push:
+    branches:
+      - master
+    paths:
+      - .github
+      - plugins/GSdx
+  pull_request:
+    branches:
+      - master
+    paths:
+      - .github
+      - plugins/GSdx
+jobs:
+  check:
+    runs-on: ubuntu-20.04
+    strategy:
```

You might want `fail-fast: false`
```suggestion
    strategy:
    fail-fast: false
```